src/transformers/models/fsmt/modeling_fsmt.py

```diff
@@ -597,10 +614,14 @@ def forward(
         x = nn.functional.dropout(x, p=self.dropout, training=self.training)
         x = residual + x
         x = self.self_attn_layer_norm(x)
+        if not self.normalize_before:
+            x = self.self_attn_layer_norm(x)
```

Comment:
this looks like an editing ommision, one line up (616) should be deleted then as you already have it under `if`

Reply:
Thanks for finding the editing ommision! I also edited that part :)